Framework/PythonInterface/mantid/plots/legend.py

```diff
@@ -127,6 +127,10 @@ def from_view_advanced(cls, view):
 
     @classmethod
     def create_legend(cls, props, ax):
+        # For Matplotlib <3.2 we have to remove the 'None' string from the title
```

Comment:
@StephenSmith25 I meant to ask. Would it be possible to add unit test somewhere for this?

Reply:
Yeah, I can write a test somewhere for it. Should I create a separate PR for this?

Reply:
:+1: Thanks. Yeah please create a separate PR.